test/fixtures/toxiproxy.json

```diff
@@ -3,5 +3,10 @@
     "name": "semian_test_mysql",
     "upstream": "localhost:3306",
     "listen": "localhost:13306"
+  },
```

Comment:
hm this file isn't really a fixture, is it? should this be in a different directory?